src/services/export-feature/export-feature.service.ts

```diff
@@ -1,16 +1,19 @@
 import { Feature, Map } from 'ol'
+import { Geometry } from 'ol/geom'
 
 import { ExportFeatureGpx } from './export-feature-gpx'
 import { ExportFeatureKml } from './export-feature-kml'
 import { ExportFeatureShapefile } from './export-feature-shapefile'
+import { ExportFeatureProfileCsv } from './export-feature-profile-csv'
 
-export type exportFormat = 'kml' | 'gpx' | 'shapefile'
+export type TExportFormat = 'kml' | 'gpx' | 'shapefile' | 'csv'
+export type TFeatExport = Feature<Geometry> & Feature<Geometry>[]
```

Comment:
What does the `T...` stand for?

Reply:
Alright, indeed I prefer without the `T`, but this may be personal :slightly_smiling_face: